modules/core/src/main/scala/org/scalasteward/core/sbt/package.scala

```diff
@@ -49,9 +48,8 @@ package object sbt {
     val name = "StewardPlugin.scala"
     // I don't consider reading a resource as side-effect,
     // so it is OK to call `unsafeRunSync` here.
-    Resource
-      .fromAutoCloseable(IO(Source.fromResource(s"org/scalasteward/plugin/$name")))
-      .use(src => IO(FileData(name, src.mkString)))
+    io.readResource[IO](s"org/scalasteward/plugin/$name")
```

Comment:
`show`?

Reply:
I'm not a big fan of the `Show` type class and I don't think we use `show` anywhere else in the code base. If we would have an interpolator that ensures the interpolated value is a `String`, I'd use it here and probably everywhere else.

Reply:
Alright, let's keep it with `s` then :)